### DISCUSSION TITLE
How do I shut down Plotman Plot if I want to run Plotman Interactive

### DISCUSSION THREAD

**Discussion by Speeddmn2421** [ericaltendorf/plotman#637] [Q&A] [answered]
How do I shut down Plotman Plot if I want to run Plotman Interactive?

**Comment by leonidcan** [comment, marked as answer]
Check my post #573 , I asked same question but did not get answer I wanted. I just stick with interactive for now, works great and I can quick anytime, change config and start over

**Reply by Speeddmn2421** [reply]
Yes but I have Plotman Plot running now. Can I start Plotman Interactive at the same time? I thought these were independent processes. Will they conflict? I though I read something about if they both try to start a plot at the same time it will cause a problem. I have Plotman Plot running now and want to switch to Plotman Interactive. I dont fully understand how the processes interact.

**Reply by Speeddmn2421** [reply]
I really wanted to just reload the config but the only instructions on how to do that are for Plotman Interactive

**Reply by leonidcan** [reply]
I think they will conflict. You can kill plotman process.

**Reply by Speeddmn2421** [reply]
Nice. Thank you. That worked perfect. Im new to Ubuntu.
Killed the plotman process via System Manager -> Processes if anyone has a similar question in the future.
Started Plotman Interactive.
Plot info for all active plots showed up correctly in Interactive.
